CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+v1.0.2
+------
+* **Release date**: 2016-11-17.
+* **Requires:** [tileserver v1.0.0](https://github.com/mapzen/tileserver/releases/tag/v1.0.0) and [tilequeue v1.0.1](https://github.com/mapzen/tilequeue/releases/tag/v1.0.1) and [mapbox-vector-tile v1.0.0](https://pypi.python.org/pypi/mapbox-vector-tile/v1.0.0).
```

Comment:
Final `requirements` question is more existential than for this PR, we can talk about it later.